src/api/js/test.js

```diff
@@ -0,0 +1,150 @@
+const z3 = require("./build/Release/z3");
```

Comment:
should really live in an "examples/js" directory.
The dependency on "/build/release" would also go away.

Reply:
Thanks for your comments. `build/Release/z3` is the compiled add on that `test.js` is importing. Right now, it is generated inside of the `src/api/js` directory. I was thinking of the task of moving it to the right place and installing it as part of the build process as item 1 in #1791 and putting out actual examples as item 2. Can we do this incrementally in this manner? This would potentially allow other people to jump in and help out.